ios/ReactNativeAdsFacebook.xcodeproj/project.pbxproj

```diff
@@ -304,26 +302,38 @@
 		D2AFD33D211B138E00580DF0 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = W3VSH3LGL7;
-				FRAMEWORK_SEARCH_PATHS = "$(PROJECT_DIR)/../../../ios/Pods/FBAudienceNetwork/";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/**",
```

Comment:
Not good.
remove 
$(PROJECT_DIR)/** (there is no framework included) 
set `$(PROJECT_DIR)/../../../ios/Pods/FBAudienceNetwork/`

Reply:
This is for the case of compiling the library on its own while testing.